doc/classes/ScrollContainer.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ScrollContainer" inherits="Container" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A helper node for displaying scrollable elements such as lists.
+		A container used to provide a child control with scrollbars when needed.
```

Comment:
I assume you mean "it provides scrollbars to child control", but this can be misunderstood.